core/src/main/java/com/expedia/www/haystack/client/idgenerators/RandomUUIDGenerator.java

```diff
@@ -22,7 +22,12 @@
 public class RandomUUIDGenerator implements IdGenerator {
```

Comment:
i wonder if we should do the changes here or create a separate generator.

Reply:
Yes, it will be better to put a new generator class name, might be B3IdGenerator

Reply:
It's not a true b3 id generator. it's still uuids but the key difference is that the values are "convertible" to b3.
I don't know what name to use.

And now that I'm writing this I'm questioning the value of having 2 uuid generators. This one still respects the previous contract.

Reply:
How does B3 generator will look like? Can't we write a new one?

Reply:
Also if you write new one, I would also ask to write KeyConvention that uses B3-Header name. May be put it inside a different module say 'zipkin-compatiblity' 
When someone creates a tracer object, they can customize it by providing this readymade generator and keyconvention

Reply:
I can, but i'd like to do that in a separate pr if that's ok?

These changes here are just to make it possible to convert between UUIDs and B3.

Reply:
Fair enough.